subsys/mgmt/ec_host_cmd/Kconfig.logging

```diff
@@ -10,3 +10,18 @@ config EC_HOST_CMD_LOG_DBG_BUFFERS
 	  Every command is logged with the debug logging level. Use this config
 	  to decide, if full reqest and response buffers are logged alongside
 	  other command parameters.
+
```

Comment:
If you add a helper symbol, you can simplify the guards in the C code.

```
config EC_HOST_CMD_LOG_SUPPRESSED
    bool
    default y if (EC_HOST_CMD_SUPPRESSED_NUMBER > 0)
```

Then the C code uses `#ifdef CONFIG_EC_HOST_CMD_LOG_SUPPRESSED`.  Alternately, you could make the `SUPPRESSED_NUMBER` and `SUPPRESSED_INVERVAL` symbols dependent on `EC_HOST_CMD_LOG_SUPPRESSED`.

Reply:
Good point, done.